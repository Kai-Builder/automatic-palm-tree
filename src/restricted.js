import React from 'react';
import {restricted} from './restricted.scss';

export const Restricted = () => (
  <div className={restricted}>
    <div>{'Restricted user.'}</div>
  </div>
);
