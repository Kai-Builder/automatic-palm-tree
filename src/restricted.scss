.restricted {
  padding: 20px;
}
