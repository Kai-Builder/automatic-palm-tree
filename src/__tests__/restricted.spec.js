import React from 'react';
import {Restricted} from '../restricted';
import {shallow} from 'enzyme';

describe('Restricted', () => {
  const requiredProps = () => ({
  });
  const render = (props = requiredProps()) => shallow(<Restricted {...props} />);

  test('component renders', () => {
    const wrapper = render();

    expect(wrapper).toMatchSnapshot();
  });
});

