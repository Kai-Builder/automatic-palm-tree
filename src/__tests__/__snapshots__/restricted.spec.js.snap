// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restricted component renders 1`] = `
<div
  className="restricted"
>
  <div>
    Restricted user.
  </div>
</div>
`;
